composer.json

```diff
@@ -10,9 +10,12 @@
     },
     "scripts": {
         "test": "composer single && composer local && composer parallel",
-        "single": "./bin/behat --config=config/single.conf.yml",
-        "local": "./bin/behat --config=config/local.conf.yml",
-        "parallel": "CONFIG_FILE=config/parallel.conf.yml /usr/bin/env php lib/parallel.php"
+        "single": "\"./bin/behat\" --config=config/single.conf.yml",
```

Comment:
Was it not working previously?

Reply:
this previously worked on mac and linux but failed on windows